packages/os/configure-boot-mount.service

```
@@ -0,0 +1,16 @@
+[Unit]
+Description=Configure mount points for the active boot partition
+ConditionPathExists=!/proc/vmcore
+RefuseManualStart=true
+RefuseManualStop=true
+Wants=prepare-local.service
+After=prepare-local.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/prairiedog configure-boot-mount
```

Maybe this for consistency with the suggested unit name:
```suggestion
ExecStart=/usr/bin/prairiedog prepare-boot
```